graphrag/config/models/cluster_graph_config.py

```diff
@@ -14,15 +14,10 @@ class ClusterGraphConfig(BaseModel):
     max_cluster_size: int = Field(
         description="The maximum cluster size to use.", default=defs.MAX_CLUSTER_SIZE
     )
-    strategy: dict | None = Field(
-        description="The cluster strategy to use.", default=None
+    use_lcc: bool = Field(
+        description="Whether to use the largest connected component.",
+        default=defs.USE_LCC,
+    )
+    seed: int | None = Field(
+        description="The seed to use for the clustering.", default=None
```

Comment:
Should the default seed value be set here instead?

Reply:
Good idea - I think the most reasonable approach is actually that not passing any seed results in a new random one each time, and our system can set a default for own repeatability reasons.